components/layout_2020/flow/inline.rs

```diff
@@ -1305,130 +1666,36 @@ impl TextRun {
-                                          glyphs: Vec<std::sync::Arc<GlyphStore>>,
-                                          inline_advance| {
-            if glyphs.is_empty() {
-                return;
-            }
-
-            let last_whitespace_advance = match (white_space.preserve_spaces(), glyphs.last()) {
-                (false, Some(last_glyph)) if last_glyph.is_whitespace() => {
-                    last_glyph.total_advance()
-                },
-                _ => Au::zero(),
-            };
-
-            ifc.push_line_item(
-                inline_advance,
-                LineItem::TextRun(TextRunLineItem {
-                    text: glyphs,
-                    base_fragment_info: self.base_fragment_info.into(),
-                    parent_style: self.parent_style.clone(),
-                    font_metrics,
-                    font_key,
-                    text_decoration_line: ifc.current_inline_container_state().text_decoration_line,
-                }),
-                Length::from(last_whitespace_advance),
-            );
-        };
+        for (run_index, run) in runs.into_iter().enumerate() {
+            ifc.possibly_flush_deferred_forced_line_break();
 
-        let line_height = line_height(&self.parent_style, &font_metrics);
-        let new_max_height_of_line = ifc.current_line_max_block_size().max(line_height);
-
-        let mut glyphs = vec![];
-        let mut text_run_inline_size = Length::zero();
-        let mut iterator = runs.iter().enumerate();
-        while let Some((run_index, run)) = iterator.next() {
-            if ifc.linebreak_before_new_content {
-                ifc.finish_current_line_and_reset(layout_context);
-                text_run_inline_size = Length::zero();
-            }
-
-            // If this whitespace forces a line break, finish the line and reset everything.
-            if self.glyph_run_is_whitespace_ending_with_preserved_newline(run) {
-                add_glyphs_to_current_line(ifc, glyphs.drain(..).collect(), text_run_inline_size);
-
-                // We need to ensure that the appropriate space for a linebox is created even if there
-                // was no other content on this line. We mark the line as having content (needing a
-                // advance) and having at least the height associated with this nesting of inline boxes.
-                ifc.current_line.has_content = true;
-                ifc.current_line
-                    .max_block_size
-                    .max_assign(new_max_height_of_line);
-
-                // Defer the actual line break until we've cleared all ending inline boxes.
-                ifc.linebreak_before_new_content = true;
-
-                continue;
-            }
+            //{
+            //    let text = &self.text[run.range.begin().to_usize()..run.range.end().to_usize()];
+            //    println!("Processing: '{text:?}'");
+            //}
```

Comment:
This should be removed

Reply:
Oops! Removed in the latest version of this change.